### PR TITLE
Correct some device settings doc issues

### PR DESCRIPTION
**Pull request by r41d**
Several categories were left out of the listings.
Also made some Bluetooth doc improvements.

### CODE REVIEW

**Review by garthvh — COMMENTED** [review]
Looks good, left one comment

File: docs/settings/config/index.mdx [hunk 1]
```diff
@@ -5,16 +5,17 @@ sidebar_label: Config Sections
 sidebar_position: 1
 ---
 
-There are six config sections in the Meshtastic firmware, these are broken out so they can be sent as small admin messages over the mesh.
+There are several config sections in the Meshtastic firmware, these are broken out so they can be sent as small admin messages over the mesh.
 
-Default settings values are prefered whenever possible as they consume no bandwidth when sent over the mesh.
+Default settings values are preferred whenever possible as they consume no bandwidth when sent over the mesh.
```
[the remark this answers is not in the document]
Comment (r41d) [reply docs/settings/config/index.mdx:10]: like, throw out the whole sentence?

[the remark this answers is not in the document]
Comment (garthvh) [reply docs/settings/config/index.mdx:10]: Yes

**Review by garthvh — APPROVED** [review]
Thanks for all the updates, ping me on discord and I will give your account flair

**Pull request merged**